.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches: [master]
```

Comment:
Useful to use `workflow_dispatch` so we can manually trigger the workflow from the repo's Actions tab if needed

Reply:
Another possibly useful thing to note @datamel, the `on: push: branches: [master]` happens when you merge PRs too, e.g. when this PR itself was just merged https://github.com/cylc/cylc-uiserver/actions/runs/633233211